tests/utilities/test_fetching.py

```diff
@@ -392,3 +392,58 @@ def __init__(self) -> None:
     m = InvalidModel()
     with pytest.raises(MisconfigurationException, match="is incompatible with `truncated_bptt_steps > 0`."):
         trainer.fit(m)
+
+
+@RunIf(min_gpus=1)
+def test_transfer_hooks_with_unpacking(tmpdir):
+
+    """This test asserts the `transfer_batch` hooks are called only once per batch and data are properly moved to
+    gpu."""
```

Comment:
```suggestion
    """This test asserts the `transfer_batch` hooks are called only once per batch."""
```
this is enough I guess. there are no assertions to test whether the data is moved to gpu or not.

Reply:
Done !